_layouts/default.html

```diff
@@ -31,8 +31,10 @@
         </div>
         <div class="collapse navbar-collapse js-navbar-collapse" style="margin-top: 20px">
           <ul class="navbar-nav nav">
-            <li><a href="{{ site.baseurl }}/docs/">Documentation</a></li>
-            <li><a href="{{ site.baseurl }}/download/">Download</a></li>
+            <li><a href="{{ site.baseurl }}/release/">Releases</a></li>
```

Comment:
I don't think a link directly to the latest download would be appropriate unless it is accompanied by instructions on verifying the release and downloading hashes/sigs. We can do this on the releases posts, but an independent link directly to the download is more problematic. Maybe a link directly to the "Latest Release" instead?